tests/functional/hold-release/17-hold-after-point/reference.log

```diff
@@ -1,5 +1,5 @@
-2015-06-03T11:33:34Z INFO - Initial point: 20100101T0000Z
-2015-06-03T11:33:34Z INFO - Final point: 20100110T0000Z
-2015-06-03T11:33:34Z INFO - [stopper.20100101T0000Z] -triggered off []
-2015-06-03T11:33:34Z INFO - [foo.20100101T0000Z] -triggered off []
-2015-06-03T11:33:42Z INFO - [foo.20100102T0000Z] -triggered off ['foo.20100101T0000Z']
+Initial point: 20100101T0000Z
+Final point: 20100110T0000Z
+[stopper.20100101T0000Z] -triggered off []
+[foo.20100101T0000Z] -triggered off ['foo.20091231T0000Z']
```

Comment:
Just to check, the reference logs now list pre-initial tasks as `triggered off` values?

Reply:
Yes. Which is fine, as reference logs aren’t user facing. And (arguably at least) it’s just as valid as ignoring them.

Reply:
Happy with this, makes more sense IMO.